lib/_tls_common.js

```diff
@@ -70,15 +70,15 @@ function SecureContext(secureProtocol, secureOptions, minVersion, maxVersion) {
   if (secureOptions) this.context.setOptions(secureOptions);
 }
 
-function validateKeyCert(name, value) {
+const validateKeyCert = (name, value) => {
```

Comment:
hmm, I am not sure about this change...supposedly v8 is able to infer the function name from the assignment but I'd prefer we just leave this as it this..